src/Core/System/SalesChannel/Event/SalesChannelContextPermissionsChangedEvent.php

```diff
@@ -11,12 +11,7 @@
 #[Package('framework')]
 class SalesChannelContextPermissionsChangedEvent extends NestedEvent implements ShopwareSalesChannelEvent
 {
-    /**
-     * @var array
-     *
-     * @deprecated tag:v6.7.0 - Will be natively typed
-     */
-    protected $permissions = [];
+    protected array $permissions = [];
```

Comment:
array annotation missing

Reply:
i checked the code and it might be that this array is from type `array<string, bool>` but i kept it like before because i was not sure if this is not a break since this property did not have any specification before

Reply:
@mitelg any opinion here?